src/v2/server.ts

```diff
@@ -56,15 +55,15 @@ app.get(
       }
 
       const headTagsString = ReactDOM.renderToString(headTags as any)
-      const sharifyData = res.locals.sharify.script()
+      const sharify = res.locals.sharify
```

Comment:
Use the sharify instance from `res.locals`, instead of importing it again.

Reply:
I couldn't find tests for v2 server. Is there a good place to test this?

Reply:
Tests are typically in the closest `__tests__` folder; if there's not a test there then it likely doesn't exist.